src/test/java/lotto/domain/LottoShopTest.java

```diff
@@ -1,29 +1,55 @@
 package lotto.domain;
 
-import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
 
 import java.util.Arrays;
-import java.util.LinkedHashSet;
-import java.util.Set;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LottoShopTest {
 
+    private static final int LOTTO_PRICE = 1_000;
 
-    @DisplayName("로또 구매 테스트")
     @ParameterizedTest
-    @CsvSource(value = {"2000:2", "15000:15", "5000:5"}, delimiter = ':')
-    public void purchaseLottoTest(int purchasePrice, int expected) {
+    @CsvSource(value = {"5000:3", "3000:1", "12000:6"}, delimiter = ':')
+    public void purchaseTicketTest(int purchasePrice, int manualLottoQuantity) {
+        //Given
+        LottoShop shop = new LottoShop(new LottoAutoMachine());
 
-        //Given & When
-        LottoShop shop = new LottoShop();
-        Lottos lottos = shop.purchase(LottoPrice.from(purchasePrice), new LottoAutoMachine());
+        //When
+        LottoTicket ticket = shop.purchaseTicket(Money.from(purchasePrice), manualLottoQuantity);
 
         //Then
-        assertThat(lottos.quantity()).isEqualTo(expected);
+        assertThat(ticket).isEqualTo(LottoTicket.of(purchasePrice / LOTTO_PRICE, manualLottoQuantity));
```

Comment:
해당 부분도 도메인 로직에 있는 상수를 사용하면 좋을 것 같아요! 😄 
```suggestion
        assertThat(ticket).isEqualTo(LottoTicket.of(purchasePrice / LottoShop.LOTTO_PRICE, manualLottoQuantity));
```